Update Park_Rules.txt (#101)
Park-Station/Parkstation: Resources/Server Info/Park_Rules.txt

> [color=#ff0000]Players who are suspected of being under age will be banned. As long as everyone can act maturely, they're welcome on the server.[/color]

Reviewer: ```suggestion
[color=#ff0000]Players who are suspected of being under age will be banned. As long as everyone can act maturely, they're welcome in the server.[/color]
```
is a home :)

[color=#ff0000]DISCONNECTING FROM OR IGNORING/EVADING COMMUNICATION FROM ADMINS WILL RESULT IN AN APPEAL ONLY BAN. The job gets really hard when you refuse to talk to the Admins, just come to our Discord and talk it out, hurt feelings will not be held.[/color]

[color=#ff0000]This is the only source of server rules that apply to Parkstation, which ideally has all the information any regular player should need. Please do not ever hesitate to ask either an Admin in[/color] [color=#DC143C]AHelp[/color][color=#ff0000] (F1) or in the Discord server if you ever want clarification on the rules. Chances are, if you were confused, we'll update them.[/color]

> We do not have a wiki, instead, we have or will have all the needed information available via Guidebooks (NumPad0), such as Space Law and Standard Operating Procedure. If we do not have some bit of information (how to do a job or how specifically to execute something a document says) you may ask an admin via [color=#DC143C]AHelp[/color] (F1) or ask other players via the [color=#66bbff]OOC[/color] chat channel.

Reviewer: ```suggestion
We do not have a wiki, instead, we have or will have all the needed information available via Guidebooks (NumPad0), such as Space Law and Standard Operating Procedure. If we do not have some bit of information (how to do a job or how specifically to execute something a document says) you may ask an admin via [color=#DC143C]AHelp[/color] (F1) or ask other players via the [color=#66bbff]OOC[/color] or [color=#66e0e0]LOOC[/color] chat channels.
```
I don't know the LOOC colour :P

[color=#ff0000]MEDIUM ROLEPLAY[/color]

> The overall intention for this server is to do MRP right. NRP behavior will be dealt with by Admins, and we will have a generally high expectation of players in terms of role-play. Note that this does not mean we expect you to memorize pages of lore, or to be a savant at improv and writing. The line drawn is we expect players to put in [color=#ffd700]effort[/color] and to be here with the mutual goal of having a fun time and telling an engaging story since that's what we believe this game to be all about.

Reviewer: ```suggestion
The overall intention for this server is to do MRP right. NRP behavior will be dealt with by Admins, and we will have a generally high expectation of players in terms of role-play. Note that this does not mean we expect you to memorize pages of lore, or to be a savant at improv and writing. The line drawn is we expect players to put in [color=#ffd700]effort[/color] and to be here with the mutual goal of having a fun time and telling an engaging story, as that's what we believe this game to be all about.
```
Fixing my own errors in your PR :P

[color=#a4885c]0.[/color] [color=#a4885c]The[/color] [color=#ffd700]Golden[/color] [color=#a4885c]Rule.[/color] Admins may exercise discretion with rules as they see fit. If you rule lawyer or line skirt, you will get removed. Admins will answer for use of this privilege.

> [color=#ff0000]ZERO TOLERANCE RULES[/color]

Reviewer: Can we just drop this overall? Like, what, the other rules are only kinda enforced?

[color=#a4885c]1.[/color] Absolutely no hate speech, bigotry, intolerance, or anything remotely similar.

[color=#a4885c]2.[/color] No engaging in explicit sexual acts.

[color=#a4885c]3.[/color] Any RP had with other players should be [color=#DC143C]entirely[/color] consensual. There are no firm guidelines on appropriate conduct, but the [color=#66bbff]OOC[/color] and [color=#66e0e0]LOOC[/color] channels are freely available. If you make another player uncomfortable, within reason, you will be held accountable.

[color=#a4885c]4.[/color] Don't communicate in-game/in-character information through methods outside of the game (such as talking in Discord with other users actively playing about the game or by talking to your sibling across the room while you are both playing). This is referred to as "Metacomming" and it is strictly forbidden.

[color=#a4885c]5.[/color] Attempting to evade game bans will result in an automatic appeal-only permanent ban. Attempting to evade job bans will result in an appeal-only permanent ban.

[color=#ff0000]GENERAL ETIQUETTE[/color]

> [color=#a4885c]6.[/color] These are English servers. Speak only English in [color=#bbbbbb]IC[/color] and [color=#66bbff]OOC[/color].

Reviewer: ```suggestion
[color=#a4885c]6.[/color] These are English servers, and you are expected to use English when communicating through any server-related channels.
```

[color=#a4885c]7.[/color] Don't use exploits or external programs to play, gain an advantage, or disrupt the round/server. This includes auto clickers and auto-hotkey scripts.

[color=#a4885c]8.[/color] Don't "multi-key" (utilize multiple alt accounts). Users knowingly using multiple SS14 accounts will have all of their accounts banned.

[color=#a4885c]9.[/color] Do not ignore the admin help relay or abuse it by flooding it with garbage, checking for admins before stating a problem (ex: "hello?", "any admins?"), or sending messages of no substance. All admin helps are sent to the SS14 Discord, which means we see everything.

[color=#ff0000]IN GAME & ROLEPLAY RULES[/color]

[color=#a4885c]10.[/color] Pick a realistic name that could appear on a birth certificate with at least a first and last name.

>     - Names don't have to be entirely realistic, but should still make sense since this is a game set quite far in the "future". Extra leeway beyond this is given to Clowns, Mimes, and non-human races.

Reviewer: ```suggestion
    - Names don't have to be entirely realistic, but should still make sense, as this is a game set quite far in the "future". Extra leeway beyond this is given to Clowns, Mimes, and non-human races.
```

>     - Names of notable famous or fictional persons or names that resemble/parody them are strictly forbidden. You are not clever if you slightly change a famous name around. Terrible names will allow admins to smite you, politely warn you, or instantly ban you, depending on severity.

Reviewer: ```suggestion
    - Names of notable famous or fictional persons or names that resemble/parody them are strictly forbidden. Terrible names will allow admins to smite you, politely warn you, or instantly ban you, depending on severity.
```

    - Names that are LRP/NRP or straight up inappropriate, such as "Holden-Hisdick" will likely get you the instant ban mentioned above.

[color=#a4885c]11.[/color] Act like an actual human being on a space station. Do not use text speak or emoticons [color=#bbbbbb]IC[/color], and do not refer to [color=#66bbff]OOC[/color] things like admins in-game. Remember that this is an MRP environment, and people are expected to try and react to situations realistically, even if it's not the "optimal" thing to do. What's good for Roleplay > What's good for Gameplay.

[color=#a4885c]12.[/color] Don't be a dick. You are playing a multiplayer game with other people who also want to enjoy the game.
    - Do not intentionally make other players' lives hell for your own amusement.

>     - The round ends when the summary pops up on your screen. Until then, RP is still expected. Remember that just because you're wrapped up doesn't mean there's no one on the shuttle finishing their story.

Reviewer: ```suggestion
    - The round ends when the summary pops up on your screen. Until then, RP is still expected. Remember that just because you're wrapped up doesn't mean there's no one at the end of the round finishing their story.
```
Planning for future servers that might not have shuttles :smartthink:

[color=#a4885c]13.[/color] Don't harass or target players across rounds for actions in prior rounds or for actions outside of the game (this is referred to as "Metagrudging".)
    - Annoying players for [color=#bbbbbb]IC[/color] reasons in the current round is fine; doing it across rounds or as a ghost role after they kill you is [color=#DC143C]not[/color].

[color=#a4885c]14.[/color] Don't use information gained from outside your character's knowledge to gain an advantage (this is referred to as "Metagaming").

>     - Using information you gain from outside your own character (such as spectating while a ghost, through screenshots or meta comms, or other means) to your advantage is strictly forbidden. You remember all events up until you fall unconscious. If you take a ghost role, unless otherwise stated, you DO NOT REMEMBER ANYTHING from your past life. As for cloning, the line is blurry for what you can recall, but generally speaking, try to remember whatever makes the story the most interesting. Note that this almost always will [color=#DC143C]not[/color] include the person who led to your death.

Reviewer: ```suggestion
[color=#a4885c]14.[/color] Don't use information gained from outside your character's knowledge to gain an advantage (this is referred to as "Metagaming").
    - Using information you gain from outside your own character (such as spectating while a ghost, through screenshots or meta comms, or other means) to your advantage is strictly forbidden. You remember all events up until you fall unconscious. If you take a ghost role, unless otherwise stated, you DO NOT REMEMBER ANYTHING from your past life. 
    - The line is blurry for cloning in regards to what you can recall, but generally speaking, try to remember whatever makes the story the most interesting. Note that this almost always will [color=#DC143C]not[/color] include the person who led to your death.
```

[color=#a4885c]15.[/color] Follow escalation rules and don't murder someone for slipping you, use common sense.
    - You can defend yourself to the extent of protecting your own life.

>     - [color=#6494F0]Security[/color] is unique in escalation and can escalate to less lethal to arrest criminals as needed.

Reviewer: This rules is confusing to me. Everyone is allowed to deescalate. I say we remove it and mention it in the Sec section.

If you regularly play [color=#334E6D]Command[/color] or [color=#6494F0]Security[/color] roles and got this far, I applaud you for reading.

[color=#a4885c]19.[/color] If you sign up for a [color=#334E6D]Command[/color] or [color=#6494F0]Security[/color] role, you are expected to know the basics of the game, your job, and the job(s) you supervise, if any.
    - Failure to know your job or how to play the game in general as [color=#334E6D]Command[/color] or [color=#6494F0]Security[/color] is liable to result in a job ban.
    - As a [color=#334E6D]Command[/color] or [color=#6494F0]Security[/color] role, don't make friends with antagonists and trade contraband with them.
    - Don't engage in common troublemaker behavior and lawbreaking as a [color=#334E6D]Command[/color] or [color=#6494F0]Security[/color] role.
    - Do not immediately abandon your position as a [color=#334E6D]Command[/color] role and go do whatever you want instead of managing your department/the station.
        - As a [color=#334E6D]Command[/color] role, do not take over the jobs of others. The [color=#334E6D]HoP[/color] is not the [color=#6494F0]HoS[/color], for instance, and holds no direct power over Sec.

>         - The [color=#334E6D]HoP[/color] is [color=#DC143C]not[/color] the Captain. [color=#DC143C]There is no Captain.[/color] Any [color=#334E6D]HoP[/color] or other command member who gives themself all access without reason or permission is liable for arrest and demotion.

Reviewer: ```suggestion
        - The [color=#334E6D]HoP[/color] is not the Captain. Any [color=#334E6D]HoP[/color] or other command member who gives themselves all access without reason or permission is liable for arrest and demotion.
```

> [color=#a4885c]20.[/color] [color=#6494F0]Security[/color]/[color=#334E6D]Command[/color] should try to remain non-lethal and effect arrests, except in the following special circumstances, where they may choose to use lethal force:

Reviewer: ```suggestion
[color=#a4885c]20.[/color] [color=#6494F0]Security[/color] should try to remain non-lethal and effect arrests, except in the following special circumstances, where they may choose to use lethal force:
```

    - Lethal force is used against you (ex: firearms, lasers, disabling weapons with intent to kill, deadly melee weapons). Note your first option should still be to stun, but if your life is in danger, you can do what you must.
    - You determine that your life or the life of an innocent is in immediate danger

>     - The suspect is unable to be safely detained by less-lethal means

Reviewer: This feels like "I ran out of disabler shots" baiting.

    - If no other reasonable options are readily available and allowing the suspect to continue would be an unreasonable danger to the station/crew

> [color=#6494F0]Security[/color]/[color=#334E6D]Command[/color] will be expected to answer for use of lethal force. [color=#6494F0]Security[/color]/[color=#334E6D]Command[/color] will be expected to effect arrests on criminals and prevent them from dying while in custody, even if lethal force is used. [color=#6494F0]Security[/color]/[color=#334E6D]Command[/color] is strongly encouraged, but not required, to clone antagonists and effect a permabrigging or other sentence as deemed appropriate.

Reviewer: ```suggestion
[color=#6494F0]Security[/color] will be expected to answer for use of lethal force. [color=#6494F0]Security[/color] will be expected to effect arrests on criminals and prevent them from dying while in custody, even if lethal force is used. [color=#6494F0]Security[/color] is strongly encouraged, but not required, to clone antagonists and effect a permabrigging or other sentence as deemed appropriate.
```

> [color=#a4885c]21.[/color] [color=#6494F0]Security[/color]/[color=#334E6D]Command[/color] are expected to protect detainees in their custody to the best of their ability so as long as it does not come to an unreasonable risk to themselves, the crew, or the station at large to do so.

Reviewer: ```suggestion
[color=#a4885c]21.[/color] [color=#6494F0]Security[/color] are expected to protect detainees in their custody to the best of their ability so as long as it does not come to an unreasonable risk to themselves, the crew, or the station at large to do so.
```

[color=#ff0000]IN GAME & ROLEPLAY RULES[/color]

[color=#a4885c]9.[/color] Pick a realistic name that could appear on a birth certificate with at least a first and last name (leeway is given to this for Clowns, Mimes, and non-human races).

>     - Names of notable famous or fictional persons or names that resemble/parody them are strictly forbidden. You are not clever if you slightly change a famous name around. Terrible names open you up to being politely reminded to change it, smote, or instantly banned, depending on severity.

Reviewer 2: technically the correct past participle here is "smitten" but given the more common usage of that word is an adjective meaning struck by affection or longing, it may be better to reword this entirely or use "smited" since it sounds more correct while retaining unambiguity

Author: who are you

Reviewer 2: i ask myself this constantly

Author: ok, I'll rewrite it for clarity since I dislike the alternate forms of 'smite'
